server/src/main/java/org/opensearch/action/admin/cluster/remotestore/restore/RestoreRemoteStoreRequest.java

```diff
@@ -130,7 +130,7 @@ public RestoreRemoteStoreRequest source(Map<String, Object> source) {
             String name = entry.getKey();
             if (name.equals("indices")) {
                 if (entry.getValue() instanceof String) {
-                    indices(Strings.splitStringByCommaToArray((String) entry.getValue()));
+                    indices(org.opensearch.core.common.Strings.splitStringByCommaToArray((String) entry.getValue()));
```

Comment:
Any restrictions not to use `import org.opensearch.core.common.Strings;`?

Reply:
Yeah, the `toString` method below calls the `Strings.toString(XContentType` method which can't be used yet. This will be refactored and simplified in a follow up PR.